docs/blitz/design/actions/actions_list.rst

```diff
@@ -404,6 +404,53 @@ snapshot you can specify to not delete it. See `example` below.
         device: my_device
         delete_snapshot: False #optional, default is True
 
+yang_snapshot
+^^^^^^^^^^^^^
+
+The ``yang_snapshot`` action is another alternative to capture related device
+configuration in a lightweight way. It does not save all device configuration,
+instead, it scans Genie Trigger information and find out ``yang`` actions that
+will change the device configuration before the next ``yang_snapshot_restore``
+action. Note that one ``yang_snapshot`` action can be followed by one or
+multiple ``yang_snapshot_restore`` actions. When the related ``yang`` actions
+are known, ``yang_snapshot`` action is able to snapshot partial device
+configuration, which will be restored when the ``restore_config_snapshot``
+action comes.
+
+Currently the ``yang_snapshot`` action is only supported with Netconf protocol.
+More protocols will be added in the future.
+
+.. code-block:: YAML
+
+    - save_config_snapshot:
```

Comment:
Isn't this supposed to be ``- yang_snapshot``?

Reply:
Fixed.